### ISSUE TITLE
Publishing to artifactory does not work

### ISSUE THREAD

**Issue by RoRoche** [Praqma/gradle-plugin-bootstrap#6]
Hi,

Thank you for the interresting article and the demo project.

I'm applying the same configuration and try to publish on an internal artifactory with this configuration:

```groovy
artifactory {
    contextUrl = "${artifactoryHost}"
    publish {
        repository {
            repoKey = 'libs-snapshot-local'
            username = "${artifactoryUser}"
            password = "${artifactoryToken}"
            maven = true
        }
        defaults {
            publications 'pluginPublication'
            publishBuildInfo = true
            publishArtifacts = true
            publishPom = true
        }
    }
}
```

But it does not publish the "markers" and a project including it can not resolve the plugin.
Any idea, suggestion?

**Comment by praqma-thi** [comment]
Hey there,

It's been a while since I've had the opportunity to work with Gradle plugins so I can't tell off the top of my hat. Your configuration looks fine at first glance. Where/how do you define your `pluginPublication`?

**Comment by RoRoche** [comment]
Here is the configuration for `pluginPublication`:

```groovy
publishing {
    publications {
        pluginPublication(MavenPublication) {
            from components.java
            groupId project.group
            artifactId project.name
            version project.version
        }
    }
}
```

**Comment by praqma-thi** [comment]
Under the current circumstances I don't have access to an Artifactory instance, however, just playing with it locally netted me some interesting results. 

When I tried `publishToMavenLocal`, Gradle blew up stating:
`Cannot publish multiple publications with coordinates 'com.praqma:demo:1.1.0' to repository 'mavenLocal'`

Downgrading my Gradle wrapper to 5.5.1 solved this, and published the plugin just fine.

Try changing your Gradle wrapper version to 5.5.1 by calling `./gradlew wrapper --gradle-version=5.5.1`, or see if upgrading to a more recent version also fixes it.

Also, when consuming the plugin, I bumped into the issue that my blog snippets try to apply the plugin at version `1.0.0`, but the GitHub project publishes at `1.1.0`, boneheaded mistake but it happens to all of us. Make sure you're consuming the version you're publishing.
